dockerfiles/jira/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM node:8-slim
+
+MAINTAINER Snyk Ltd
+
+# Install broker
+RUN npm install --global snyk-broker
+
+# Don't run as root
+WORKDIR /home/node
+USER node
+
+# Generate default accept filter
+RUN broker init jira
+
+
+
+######################################
+# Custom Broker Client configuration #
+# Redefine in derived Dockerfile,    #
+# or provide as runtime args `-e`    #
+######################################
+
+# Your unique broker identifier
+ENV BROKER_TOKEN <broker-token>
+
+# your personal credentials to your jira
+ENV JIRA_USERNAME <username>
+ENV JIRA_PASSWORD <password>
+
+# The Jira server API URL
```

Comment:
Should read `Your Jira Server host, excluding scheme`, similar to:
https://github.com/snyk/broker/blob/master/dockerfiles/bitbucket-server/Dockerfile#L32